incompatibility/incompatibility-report/incompatibility-report.proto

```diff
@@ -0,0 +1,52 @@
+
+// Copyright 2017 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "../incompatibility-report;incompatibility";
+
+package incompatibility;
+
+message Incompatibility {
+    string token = 1;
+    string classification = 2;
+}
+
+message IncompatibilityReport {
+    repeated Incompatibility incompatibilities = 1;
+}
+
+enum Severity {
+    INFO = 0;
+    FAIL = 1;
+}
+
+message FormattedIncompatibility {
```

Comment:
I wonder if there is a more applicable name you could use?   When I see Formatted I think it's an Incompatibility that has had a string formated in some way?    Whereas this is really a more description representation of the incompatibility.

Reply:
FormattedIncompatibility -> IncomaptibilityDescription
FormattedIncomaptibilityReport -> IDReport